### ISSUE TITLE
why the prediction is far more unstable than the gt pose?

### ISSUE THREAD

**Issue by yukichou** [eth-siplab/AvatarPoser#14]
I used the pretrained model to predict the test set of AMASS dataset and visualized the prediction, but the output is severe jittering comparing to the GT pose. ( I visualized the prediction and the GT in unity, the GT pose is smooth and accurate)
Regarding to the root orientation which comes from the stabilizer, is badly shaking too. BTW, the test score is 1.98/2.51/24.11.
Does anyone know this problem?

**Comment by yukichou** [comment]
Also, I found out the ik_module is not being used in the code, while the framework in the paper is using it to optimize the arm's joints, why is that?

**Comment by jiaxi-jiang** [comment]
Hi, it seems that I uploaded a wrong pretrained model which was trained on another data split, so please just retrain the model from scratch. Sorry I haven't time to clean the code of IK yet, but it is easy to implement.
